.github/workflows/python-app.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches: [ "main" ]
```

Comment:
This will be called twice on push to main branch. Once with this, and once with the workflow call

Reply:
Ohh right, I can remove the push branches main part then, right?

Reply:
yes ;)